src/domain-function.test.ts

```diff
@@ -50,7 +50,7 @@ describe('makeDomainFunction', () => {
       envParser,
     )(async ({ id }, { uid }) => [id, uid])
 
-    expect(await handler({ id: '1' }, { uid: '2' })).toEqual({
+    expect(await handler({ id: '1' }, { uid: 2 })).toEqual({
```

Comment:
couldn't figure out why this line was changed.. mistake?

Reply:
The preprocess is not part of the type inference by `z.infer`. Which means that it cannot be relied upon to type-check the code. In practice the consumers of the library would move any environment preprocess to the call site. There seems to be very few cases such as this in our current code bases.

Reply:
Maybe I should remove the preprocess from this env to avoid having bogus examples in the tests.